<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <title>Sharing proteomics data, trickier than it seems</title>
  <meta name="description" content="Reading the blog post from Cameron Neylon about how research incentives should align with research outcomes made me see a clear relation with the problem of ...">

  <link rel="stylesheet" href="/css/main.css">
  <link rel="canonical" href="http://dannynavarro.net/2010/07/31/sharing-proteomics-data-trickier-than-it-seems/">
  <link rel="alternate" type="application/rss+xml" title="Danny Navarro's Blog" href="http://dannynavarro.net/atom.xml" />
</head>

  <body>

    <header class="site-header">

  <div class="wrapper">

    <a class="site-title" href="/">Danny Navarro's Blog</a>

    <nav class="site-nav">
      <a href="#" class="menu-icon">
        <svg viewBox="0 0 18 15">
          <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
          <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
          <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
        </svg>
      </a>
      <div class="trigger">
        
          
          <a class="page-link" href="/atom.xml">Subscribe</a>
          
        
          
        
          
        
      </div>
    </nav>

  </div>

</header>

    <div class="page-content">
      <div class="wrapper">
        <div class="post">

  <header class="post-header">
    <h1 class="post-title">Sharing proteomics data, trickier than it seems</h1>
    <p class="post-meta">Jul 31, 2010</p>
  </header>

  <article class="post-content">
    <p>Reading the <a href="http://cameronneylon.net/blog/metrics-of-use-how-to-align-researcher-incentives-with-outcomes/">blog post</a> from <a href="http://cameronneylon.net">Cameron Neylon</a> about how research incentives should align with research outcomes made me see a clear relation with the problem of sharing data in proteomics.</p>

<p>Accessing genomics or transcriptomics data is more or less straightforward compared to proteomics data. You go to a data repository and download whatever you are looking for. Anyone who tries to do something similar in proteomics usually ends up downloading spreadsheets with incomplete data from supplemental material of published articles.</p>

<p>It’s frequent to hear proteomics informaticians complaining about experimentalists not making their data easily available and how they could be so greedy of not publishing the data for what they have been funded with public money. But jumping into conclusions about experimentalists being greedy or sloppy is not fair to me. The real issue is far deeper than that.</p>

<p>First of all, in order to share proteomics data the first requirement is to have the proper infrastructure. Lately there has been some projects aiming to provide the sharing infrastructure. I would say that the most popular proteomics repositories are <a href="http://www.peptideatlas.org/">PeptideAtlas</a>, <a href="http://www.humanproteinpedia.org/">Proteinpedia</a>, <a href="http://www.ebi.ac.uk/pride/">PRIDE</a> and <a href="https://proteomecommons.org/">ProteomeCommons</a>.</p>

<p>As far as I know, the main way Proteinpedia and PeptideAtlas handle data upload is with manual curators who study the format and the way the proteomics experiments were done. Then they come up with the best way to put the data in the repository. Usually is a combination of custom parsing with some manual editing to make experiments consistent in the repository. Obviously this kind of method is not that scalable for every proteomics experiment out there. In order to cope Proteinpedia focuses on human submissions only, whereas PeptideAtlas team is already looking into other <a href="http://www.peptideatlas.org/upload/">solutions</a> to facilitate data submission.</p>

<p>PRIDE, in the other hand, accept only data submission in PRIDE XML format. The generation of this XML format is not supported by most of the proteomics tools experimentalists use. However the PRIDE team provide an <a href="http://code.google.com/p/pride-converter/">application</a> that tries to convert every format out there and every experiment design to the PRIDE XML format. But because they try cover every single aspect of every experiment, I’ve seen experimentalists struggle when trying to fill all the forms. Moreover, they also don’t like the rigidity of the model to depict what the proteomics experiment was about. It’s true that there are some optional controlled vocabulary terms to give flexibility but still experiments have difficulties wrapping their head about how to use those terms.</p>

<p>This problem is not exclusive of PRIDE. Each proteomics lab uses its own mass spec terminology and frequently forms designed by developers with no first-hand experience making experiments don’t match the terms the experimentalists would understand. After all, experimentalists like to spend their time with experiments, they don’t like spending their time in things that are considered bureaucracy. The PRIDE team is aware of this problem a keeps trying with more <a href="http://www.ebi.ac.uk/pride/proteomeharvest/">familiar ways</a> for experimentalists to fill form data.</p>

<p>ProteomeCommons seems to be the repository getting most traction. It’s currently where most proteomics experimentalists are submitting their data to fend off journal editors’ complains about the lack of published data. ProteomeCommons is built on top of the <a href="https://trancheproject.org/">Tranche</a> network, a kind of global distributed filesystem, that potentially offers infinite scalability to store data by just adding more nodes to the network. The tranche network looks just like a big hard disk with several files. Everybody can upload anything they like. That’s where ProteomeCommons comes in place, it’s the web gateway to upload the data, the web application offers some options to annotate the data, but it’s not as complete as what you have in the other databases. That’s understandable because if they bothered the experimentalists with thousands of forms with mandatory fields, the experimentalist wouldn’t submit their data to ProteomeCommons. It’s also worth mentioning that ProteomeCommons is not required to use Tranche network, any other proteomics repository could use Tranche network as a backend to store huge proteomics files. The rest of the repositories are currently looking into using the Tranche network to store the huge amounts of proteomics data, specially derived from raw data.</p>

<p>You might have notice the conspicuous absence of format standards in my description about the different repositories. If everybody used the same proteomics standards the infrastructural problems to share data would have been solved. Right?</p>

<p>The major effort to standardized proteomics formats is being carried out by <a href="http://www.psidev.info/">HUPO-PSI</a>. It’s a kind of consortium where they have regular meetings where representatives of different proteomics groups among the world agree on what has to go in the standard and how. You can follow the discussions and chip in for what would you like to have in the formats.</p>

<p>Aside of the typical problems of something <a href="http://www.codinghorror.com/blog/2005/06/the-pontiac-aztek-and-the-perils-of-design-by-committee.html">designed by committee</a>, it remains to be seen if mass spec vendors and proteomics software developers will fully embrace the standards. Proteomics data is highly heterogeneous by nature, there are very different kinds of proteomics experiments depending on what is the research being done. High quality proteomics experiments is not something that can be converted into an assembly line process where everything can be easily is fixed and standardized.</p>

<p>However the new stable <a href="http://www.psidev.info/index.php?q=node/257">format</a> <a href="http://www.psidev.info/index.php?q=node/403">releases</a> from HUPO PSI look good enough to me to at least start making the data exchange among repositories possible. There is also the promise from several mass spec vendors of future commitment to fully support the standards. I hope all those promises don’t end up in just that.</p>

<p>In my opinion all these infrastructural difficulties are going to be solved somehow relatively soon. The problem is that I don’t think that just by solving the infrastructural problems everybody will start sharing data transparently. There are other difficulties.</p>

<p>The main reason experimentalists are, at least, uploading their data to Tranche is to avoid being bugged by proteomics journals into making their proteomics data available. Many proteomics journals are getting really <a href="http://www.mcponline.org/site/misc/ParisReport_Final.xhtml">serious</a> about this making the data available. Why proteomics journals are so interested in having the authors making their data available?</p>

<p>One could argue that editors of these journals believe in the moral imperative of making the data available but I don’t buy ethics as the main reason. The majority of biomedical scientific journals are still for profit companies, not academic institutions. In order to survive they have to make money selling something as every other company. For a journal publishing articles with lots of citations from other journals, with lots citations themselves, are the best way to guarantee that companies and institutions will keep renewing the yearly subscriptions.</p>

<p>It’s not something that I can’t demonstrate with facts, but lately I’m getting the feeling that proteomics is being disregarded by people in other biological fields as <em>low quality research</em>. After all proteomics is just a technology, a tool, to find out biological insights. Mass spec research by itself wouldn’t get so much funding if it couldn’t be used for biological research. What I think it’s happening is that biologists are taking less seriously biological findings in pure proteomics journals. Most published proteomics experiments are irreproducible and if you start digging into the published data you frequently find many false positives. That’s why proteomics journals editors are enforcing the experimentalists to release their data and make it as transparent as possible. They hope they can gain more credibility and get more citations from non-proteomics journals.</p>

<p>But still one can see that most experimentalists are reluctant to make their best data fully available. Many informaticians trying to analyze the data think that this attitude is because of cultural resistance to change. Many of these informaticians try to evangelize the experimentalists about why is so important to share data. Among evangelists the most notable group is the <a href="http://www.fixingproteomics.org/">Fix Proteomics Campaign</a>, which proposes some habits to make proteomics more credible.</p>

<p>But experimentalists are not dummies. I have seen them changing really quickly any habit if they find something better. The problem is that making their data transparently available is worse for them and here is where I think the campaigns miss the point. Let me explain something unique about mass spectrometry proteomics that seems easily forgotten by many people.</p>

<p>Mass spectrometers are really expensive instruments. Getting the adequate skills to operate them takes several years of training. To make things more costly these instruments become obsolete in a matter of few years because there new ones are constantly new ones coming up with better features. When a new instrument arrives to the lab a lot of time is spent optimizing it and learning how to troubleshoot it. If you don’t keep getting those new mass spectrometers you are left behind by the competitors because they can get advantage of more powerful instruments.</p>

<p>How in academics is possible to maintain a high funding inflow? In a company you have to sell a a service or a product but you can’t do so in an academic group. Usually you rely on grant agencies to provide funding. Granting agencies grant money by scientific productivity of the group. Publications in <em>reputable</em> journals are the main tangible measurement used by granting agencies for scientific productivity.</p>

<p>But most journals, as I said before, have to operate like companies. Proteomics data by itself is not publishable, if there is no story with some biological insight or some novel way to improve results, what will you write in a proteomics paper with just high quality data? Generating high quality proteomics data is damn difficult, I would say even more difficult than to come with fancy analysis of data. Let me explain.</p>

<p>People coming from genomics and transcriptomics fields sometimes forget that the chemical nature of proteins is much more diverse than DNA or RNA. After all, DNA and RNA have more or less homogeneous chemical properties regardless of its sequence. Proteins, in the other hand, are chemically completely different from each other depending on the sequence - that’s why they can carry out so many molecular functions. A proteins from the nucleus are completely different than the proteins from the cytoplasmic membrane.</p>

<p>The proteome is also much more dynamic than the genome. The same cell under different conditions show completely different proteome profile. You also have to take chemical modifications of proteins into account, which are only detectable by probing the proteins directly. Chemical modifications like phosphorylation act as functional switches for proteins, a protein with a modification has also different chemical properties than the same protein without modification. The heterogeneity of proteins makes protein purification, proper separation, and identification by mass spec an entire field by itself.</p>

<p>So an experimentalist, who has had years of training just to be able to identify proteins and chemical modifications, might, understandably, lack the skills for sophisticated analysis that will make the story sexier for proteomics journals. Software to analyze proteomics data as a <em><a href="http://en.wikipedia.org/wiki/Power_user">power user</a></em><em>, </em>without programming knowledge, is still in the early days. As a developer I can see how difficult it is to make analysis software that covers every kind of proteomics experiment with a <em>point and click</em> interface.</p>

<p>The most logical step for an experimentalist when having good data would be to look for people specialized in analysis to come up with a powerful story. Usually the best people analyzing are independent proteomics informaticians that would do the analysis only if they get the credit for it. After all they have to also have to get funded to keep doing research and they can always claim they are the ones writing the paper. But even if proteomics informaticians give proper credit to the data generators in their papers - which I wouldn’t say it’s always true -, very few granting agencies will keep the experimental lab funded just to generate data that other people will use to write publications.</p>

<p>To tackle this problem the the most powerful experimental proteomics labs are trying to aggressively hire programmers who can do analysis <em>in-house</em> so that the credit remains within the group. The first problem these groups face is that there are almost no proteomics informaticians in the job market. They have to invest in people with programming skills who will eventually get the proteomics knowledge necessary to make useful programs for analysis or to be able to analyze the data themselves.</p>

<p>Also, because of the lack of programming knowledge of the experimentalists it can be tricky for them to envision which potential programmers will have the skills required to become a good analyzer, not to mention how to motivate the programmers with an excellent about why to join their field. There are also programmers, where I include myself, that precisely look actively for this highly experimental labs instead of pure informatics groups in order to get a closer understanding of experimental data by interacting directly with the experimentalists.</p>

<p>But this kind of setting where data generators and informaticians try to work together is propitious to get into <a href="http://en.wikipedia.org/wiki/Dilbert">Dilbert-kind</a> of situations. I would say that it’s mainly because of the misunderstanding generated by the technological gap.</p>

<p>I feel fortunate of working in <a href="http://bioms.chem.uu.nl/">my current group</a> because I think it’s one of the few experimental proteomics lab where people are aware of this problem and actively try to improve the communication with the informaticians.</p>

<p>The final point I want to make in this post is that if the data generators were rewarded properly for what they are good at, generating unbeatable high quality, sharing proteomics data transparently, and as soon as it’s generated, would become mainstream. I see granting agencies rewards as the main cause for not sharing data because they usually don’t reward the generation of data accordingly. They should also reward the labs that not only make the data available but make it as accessible as possible for data analyzers, so that proteomics research field would advance much faster than it’s currently doing. I acknowledge granting agencies are changing slowly for the better but there is still a long way to go.</p>

<p>But understanding how granting agencies work and what are their motivations is still something quite fuzzy to me. I think I’m still not old enough to understand the politics behind research funding.</p>

  </article>

  <a href="https://twitter.com/share" class="twitter-share-button" data-count="none" data-url="http://dannynavarro.net/2010/07/31/sharing-proteomics-data-trickier-than-it-seems/" data-text="Sharing proteomics data, trickier than it seems by @vimes656 -" data-dnt="true">Twitter</a>
  <div class="g-plus" data-action="share" data-href="http://dannynavarro.net/2010/07/31/sharing-proteomics-data-trickier-than-it-seems/" data-annotation="none" data-width="" data-size="medium"></div>
  <div id="disqus_thread"></div> <script type="text/javascript"> var disqus_shortname = 'dannynavarrosblog'; var disqus_url = 'http://dannynavarro.net/2010/07/31/sharing-proteomics-data-trickier-than-it-seems/'; var disqus_identifier = 'http://dannynavarro.net/2010/07/31/sharing-proteomics-data-trickier-than-it-seems/'; var disqus_title = 'Sharing proteomics data, trickier than it seems'; (function () { var s = document.createElement('script'); s.async = true; s.type = 'text/javascript'; s.src = '//dannynavarrosblog.disqus.com/embed.js'; (document.getElementsByTagName('HEAD')[0] || document.getElementsByTagName('BODY')[0]).appendChild(s); }()); </script> <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript" rel="nofollow">comments powered by Disqus.</a></noscript>

</div>

      </div>
    </div>

    <footer class="site-footer">

  <div class="wrapper">

    <h2 class="footer-heading">Danny Navarro's Blog</h2>

    <div class="footer-col-wrapper">
      <div class="footer-col  footer-col-1">
        <ul class="contact-list">
          <li><a href="mailto:j@dannynavarro.net">j@dannynavarro.net</a></li>
        </ul>
      </div>

      <div class="footer-col  footer-col-2">
        <ul class="social-media-list">
          
          <li>
            <a href="https://github.com/jdnavarro">
              <span class="icon  icon--github">
                <svg viewBox="0 0 16 16">
                  <path fill="#828282" d="M7.999,0.431c-4.285,0-7.76,3.474-7.76,7.761 c0,3.428,2.223,6.337,5.307,7.363c0.388,0.071,0.53-0.168,0.53-0.374c0-0.184-0.007-0.672-0.01-1.32 c-2.159,0.469-2.614-1.04-2.614-1.04c-0.353-0.896-0.862-1.135-0.862-1.135c-0.705-0.481,0.053-0.472,0.053-0.472 c0.779,0.055,1.189,0.8,1.189,0.8c0.692,1.186,1.816,0.843,2.258,0.645c0.071-0.502,0.271-0.843,0.493-1.037 C4.86,11.425,3.049,10.76,3.049,7.786c0-0.847,0.302-1.54,0.799-2.082C3.768,5.507,3.501,4.718,3.924,3.65 c0,0,0.652-0.209,2.134,0.796C6.677,4.273,7.34,4.187,8,4.184c0.659,0.003,1.323,0.089,1.943,0.261 c1.482-1.004,2.132-0.796,2.132-0.796c0.423,1.068,0.157,1.857,0.077,2.054c0.497,0.542,0.798,1.235,0.798,2.082 c0,2.981-1.814,3.637-3.543,3.829c0.279,0.24,0.527,0.713,0.527,1.437c0,1.037-0.01,1.874-0.01,2.129 c0,0.208,0.14,0.449,0.534,0.373c3.081-1.028,5.302-3.935,5.302-7.362C15.76,3.906,12.285,0.431,7.999,0.431z"/>
                </svg>
              </span>

              <span class="username">jdnavarro</span>
            </a>
          </li>
          

          
          <li>
            <a href="https://twitter.com/vimes656">
              <span class="icon  icon--twitter">
                <svg viewBox="0 0 16 16">
                  <path fill="#828282" d="M15.969,3.058c-0.586,0.26-1.217,0.436-1.878,0.515c0.675-0.405,1.194-1.045,1.438-1.809
                  c-0.632,0.375-1.332,0.647-2.076,0.793c-0.596-0.636-1.446-1.033-2.387-1.033c-1.806,0-3.27,1.464-3.27,3.27 c0,0.256,0.029,0.506,0.085,0.745C5.163,5.404,2.753,4.102,1.14,2.124C0.859,2.607,0.698,3.168,0.698,3.767 c0,1.134,0.577,2.135,1.455,2.722C1.616,6.472,1.112,6.325,0.671,6.08c0,0.014,0,0.027,0,0.041c0,1.584,1.127,2.906,2.623,3.206 C3.02,9.402,2.731,9.442,2.433,9.442c-0.211,0-0.416-0.021-0.615-0.059c0.416,1.299,1.624,2.245,3.055,2.271 c-1.119,0.877-2.529,1.4-4.061,1.4c-0.264,0-0.524-0.015-0.78-0.046c1.447,0.928,3.166,1.469,5.013,1.469 c6.015,0,9.304-4.983,9.304-9.304c0-0.142-0.003-0.283-0.009-0.423C14.976,4.29,15.531,3.714,15.969,3.058z"/>
                </svg>
              </span>

              <span class="username">vimes656</span>
            </a>
          </li>
          
        </ul>
      </div>

      <div class="footer-col  footer-col-3">
        <p class="text">Mostly Haskell ramblings</p>
      </div>
    </div>

  </div>

</footer>

  <link href='/stylesheets/all-904ee73bdde50735aa652b14aabe03b5.css' media='all' rel='stylesheet' type='text/css'>
  <script async src='/javascripts/all-d260facd09f87a3424e14659736f1be9.js'></script>
  <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>
  <script type="text/javascript"> (function() { var po = document.createElement('script'); po.type = 'text/javascript'; po.async = true; po.src = 'https://apis.google.com/js/platform.js'; var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(po, s); })(); </script>

  </body>

</html>
